app/controllers/org_admin/questions_controller.rb

```diff
@@ -0,0 +1,213 @@
+    def index
+      authorize Question.new
+      section = Section.includes(:questions, phase: :template).find(params[:section_id])
+      edit = (current_user.can_modify_templates?  &&  (section.phase.template.org_id == current_user.org_id))
+# TODO: refactor so we're only sending back what is necessary
+      render partial: 'index', 
+        locals: { 
+          template: section.phase.template, 
+          phase: section.phase, 
+          section: section,
+          questions: section.questions, 
+          current_tab: params[:r] || 'all-templates',
+          edit: edit 
+        }
+    end
+
+    # GET /org_admin/templates/[:template_id]/phases/[:phase_id]/sections/[:section_id]/questions/[:id]
+    def show
+      question = Question.includes(:annotations, :question_options).find(params[:id])
+      authorize question
+      render partial: 'show', 
+        locals: { 
+          template: question.section.phase.template, 
+          phase: question.section.phase, 
+          section: question.section,
+          question: question, 
+          original_org: question.section.phase.template.base_org, 
+          edit: false 
+        }
+    end
+
+    # GET /org_admin/templates/[:template_id]/phases/[:phase_id]/sections/[:section_id]/questions/[:id]/edit
+    def edit
+      question = Question.includes(:annotations, :question_options, section: { phase: :template }).find(params[:id])
+      authorize question
+# TODO: refactor so we're only sending back what is necessary
+      render partial: 'edit', 
+        locals: { 
+          template: question.section.phase.template, 
+          phase: question.section.phase, 
+          section: question.section,
+          question: question,
+          current_tab: params[:r] || 'all-templates',
+          edit: true
+        }
+    end
+
+    # GET /org_admin/templates/[:template_id]/phases/[:phase_id]/sections/[:section_id]/questions/new
+    def new
+      section = Section.includes(phase: :template).find(params[:section_id])
+      question = Question.new(section: section, number: (section.questions.length > 0 ? section.questions.max{ |a, b| a.number <=> b.number }.number+1 : 1))
+      authorize question
+# TODO: refactor so we're only sending back what is necessary
+      render partial: 'new', 
+        locals: { 
+          template: section.phase.template,
+          phase: section.phase,
+          section: section,
+          question: question,
+          current_tab: params[:r] || 'all-templates'
+        }
+    end
+    
+    # POST /org_admin/templates/[:template_id]/phases/[:phase_id]/sections/[:section_id]/questions
+    def create
+      section = Section.includes(:questions, phase: :template).find(params[:section_id])
+      question = Question.new(question_params.merge({ section_id: section.id }))
+      current_tab = params[:r] || 'all-templates'
+      authorize question
```

Comment:
Please, change the order of these instructions to be something like this:

```
question = Question.new(question_params)
authorize question
section = Section.includes(:questions, phase: :template).find(params[:section_id])
current_tab = params[:r] || 'all-templates'
```
so that we don't run three queries four queries to retrieve section, questions, phase and template until pundit returns true